src/tools/illink/src/linker/Linker.Steps/BodySubstitutionParser.cs

```diff
@@ -114,11 +114,6 @@ protected override void ProcessField (TypeDefinition type, XPathNavigator fieldN
 			}
 
 			_substitutionInfo.SetFieldValue (field, res);
-
-			string init = GetAttribute (fieldNav, "initialize");
-			if (init?.ToLowerInvariant () == "true") {
-				_substitutionInfo.SetFieldInit (field);
-			}
```

Comment:
This is effectively a silent break - I would prefer we report a warning.
Pros - we don't need to "Educate" anyone on this really - the tooling will do the education for us
Cons - code to keep around and delete a release or two later (and the test)

Reply:
That works - warning is a bit more "pleasant" (can be ignored), hard crash means that the code MUST be fixed, so if this comes from a NuGet I'm screwed. But given the rarity of this, it may be fine.